coremltools/converters/mil/frontend/torch/utils.py

```diff
@@ -26,16 +26,22 @@
     1: torch.int8,
     2: torch.int16,
     3: torch.int32,
-    4: torch.int32,
     5: torch.float16,
     6: torch.float32,
-    7: torch.float32,
```

Comment:
Similarly, we should keep this as `7: torch.float64`

Reply:
Changes made in #2171